lxd/storage/backend_lxd.go

```diff
@@ -2547,8 +2548,8 @@ func (b *lxdBackend) UpdateCustomVolume(projectName string, volName string, newD
 
 // UpdateCustomVolumeSnapshot updates the description of a custom volume snapshot.
 // Volume config is not allowd to be updated and will return an error.
-func (b *lxdBackend) UpdateCustomVolumeSnapshot(projectName string, volName string, newDesc string, newConfig map[string]string, op *operations.Operation) error {
-	logger := logging.AddContext(b.logger, log.Ctx{"project": projectName, "volName": volName, "newDesc": newDesc, "newConfig": newConfig})
+func (b *lxdBackend) UpdateCustomVolumeSnapshot(projectName string, volName string, newDesc string, newConfig map[string]string, newExpiryDate time.Time, op *operations.Operation) error {
```

Comment:
@monstermunchkin I'd suggest calling the argument name `expiresAt` so that the argument name relates to the API field and so that it doesn't imply the expiry is only at date resolution.

Reply:
Sure, we could do that. However, it might be cleaner to just keep the names as they currently are, e.g. `CreationDate`, `ExpiryDate` etc. Again, I don't have a strong opinion on this.

Reply:
@monstermunchkin mmm, yes I suppose so, although I liked `expiresAt` as its got existing precedent so not introducing a new term, and ambiguous enough that one is forced to look at the data type rather than assume its a date from the variable name only.